src/styledButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { Interpolation, Theme } from "@emotion/react";
+import React from "react";
+import { focusStyle, match, useAppkitConfig } from ".";
+
+export type Kind = "normal" | "danger" | "happy";
```

Comment:
Thanks for the detailed explanation, I should have RTFM.

Regarding the naming: For the colors, I would stick with "happy". This keeps them in direct opposition to danger, and signals that they can be used for anything positive, not just call to actions.

And I can also see the argument for switching the button kind to "call to action". This hopefully makes it more clear to non-Tobira-apps that they should still use that kind of button instead of "normal", even if they do not define any "happy" colors.